foundations/foundation-metrics/src/main/java/io/servicecomb/foundation/metrics/MetricsServoObserverManager.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.foundation.metrics;
+
+import java.util.concurrent.TimeUnit;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.netflix.config.DynamicPropertyFactory;
+import com.netflix.servo.publish.BasicMetricFilter;
+import com.netflix.servo.publish.CounterToRateMetricTransform;
+import com.netflix.servo.publish.MetricObserver;
+import com.netflix.servo.publish.MonitorRegistryMetricPoller;
+import com.netflix.servo.publish.PollRunnable;
+import com.netflix.servo.publish.PollScheduler;
+
+import io.servicecomb.foundation.metrics.output.MetricsFileOutput;
+import io.servicecomb.foundation.metrics.output.servo.FileOutputMetricObserver;
+import io.servicecomb.foundation.metrics.output.servo.MetricsContentConvertor;
+import io.servicecomb.foundation.metrics.output.servo.MetricsContentFormatter;
+
+//manage and init ServoObservers
+@Component
+public class MetricsServoObserverManager {
```

Comment:
did not see what's the purpose of the manager.
it seems special for file?

Reply:
It use for init Servo PollScheduler,PollRunner and Observers

Reply:
change the class name to MetricsObserverInitializer and move it into io.servicecomb.foundation.metrics.output.servo package